MetadataGenerator/AppDelegate.m

```diff
@@ -9,340 +9,342 @@
 #import "AppDelegate.h"
 @import libPhoneNumberiOS;
 
-
 @interface AppDelegate ()
 @end
 
-
 @implementation AppDelegate
 
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
```

Comment:
I would suggest increasing the "ColumnLimit" to be MUCH higher (we have found 170 to be pretty good).

_(Can be specified in a .clang-format file.)_

Reply:
The only reason we go with 100 is that you can fit two screens (side by side diffs) on a standard macbook monitor without wrapping.